workflow_scripts/check-for-new-versions.py

```diff
@@ -3,7 +3,7 @@
 
 # Constants
 DIST_FOLDER = './dist'
-EXCLUDED_VERSIONS = os.environ.get('EXCLUDED_VERSIONS', 'v20.10.x,v23.0.x')
+EXCLUDED_VERSIONS = os.environ.get('EXCLUDED_VERSIONS', 'v20.10.x,v23.0.x,v26.1.x')
```

Comment:
Same question here

Reply:
Because there was a release for 26.1.5 https://github.com/moby/moby/releases/tag/v26.1.5, but there were no packages uploaded for the same at https://download.docker.com/linux/rhel/9/x86_64/stable/Packages/